src/Test.UnitTests.Sarif/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -687,7 +687,8 @@ public void InsertOptionalDataVisitor_FlattensMessageStringsInFix()
         public void InsertOptionalDataVisitor_ResolvesOriginalUriBaseIds()
         {
             string inputFileName = "InsertOptionalDataVisitor.txt";
-            string testDirectory = GetTestDirectory("InsertOptionalDataVisitor") + Path.DirectorySeparatorChar;
+            string inputFileText = GetResourceText(inputFileName);
+            string testDirectory = Path.GetDirectoryName(GetResourceDiskPath(inputFileName));
```

Comment:
>string testDirectory = Path.GetDirectoryName(GetResourceDiskPath(inputFileName)); [](http://example.com/codeflow?start=12&length=81)

testDirectory is used for originalUriBaseIds,
it should be end with a `/`, the current code Path.GetDirectoryName() will not have a `/` at the end.
Also need the `+ Path.DirectorySeparatorChar;`

Reply:
Excellent comment! If these tests pass after my mistake, that means we didn't validate the resulting SARIF! I think we should scrub all tests, potentially, to add this additional logic (wonder what the perf impact would be).